lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -700,6 +700,7 @@ void ResourceMap::loadDefaultConfiguration()
   addAsString("Drawable-NoSpecifiedLabel", "");
   addAsUnsignedInteger("Drawable-DataThreshold", 2000);
   addAsUnsignedInteger("Drawable-DefaultPalettePhase", 12);
+  addAsString("Drawable-DefaultPaletteName", "Tableau");
 
```

Comment:
the same entry must be added in lib/etc/openturns.conf

Reply:
Ok. Fixed